content/show/coder-radio/_index.md

```diff
@@ -12,8 +12,13 @@ active = true
 
 header_image = "/images/shows/coder-radio.png"
 
+
+support_link = "https://jupitersignal.memberful.com/checkout?plan=53334"
```

Comment:
```suggestion
support_link = "/membership"
```

Reply:
> 

No problem, I will change the style with pleasure.